<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NaviMain</class>
 <widget class="QMainWindow" name="NaviMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../navi.qrc">
    <normaloff>:/navi.png</normaloff>:/navi.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextBrowser" name="logDisplay"/>
    </item>
    <item row="1" column="0">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="latLabel">
        <property name="text">
         <string>Lat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="latEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lonLabel">
        <property name="text">
         <string>Lon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lonEdit"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="readButton">
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>483</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAgenda">
    <property name="title">
     <string>Navi</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionRestart"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help...</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionLicense"/>
   </widget>
   <addaction name="menuAgenda"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNewItem">
   <property name="text">
    <string>New Item</string>
   </property>
  </action>
  <action name="actionClearOld">
   <property name="text">
    <string>Clear Old</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionLicense">
   <property name="text">
    <string>License</string>
   </property>
  </action>
  <action name="actionRevive">
   <property name="text">
    <string>Revive Expired</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../navi.qrc"/>
 </resources>
 <connections/>
</ui>
